lib/config.py

```diff
@@ -101,59 +170,28 @@ def _setup_config_parser_options(self):
-        self.parser.add_argument('--repositories-path', '-R',
-                                 help='Directory where to clone code repositories',
-                                 default='/var/lib/host-os/repositories')
-        self.parser.add_argument('--keep-builddir',
-                                 help='Keep build directory and its logs and '
-                                 'artifacts.', action='store_true')
-        self.parser.add_argument('--mock-args',
-                                 help='Arguments passed to mock command',
-                                 default='')
-        self.parser.add_argument('--build-versions-repository-url',
-                                 help='Build versions repository URL')
-        self.parser.add_argument('--build-version',
-                                 help='Select build version from versions '
-                                 'repository')
-        self.parser.add_argument('--build-versions-repo-dir',
-                                 help='Directory to clone the build versions '
-                                 'repository',
-                                 default='./components')
         self.parser.add_argument('--log-size',
                                  help='Size in bytes above which the log file '
                                  'should rotate', type=int)
-        self.parser.add_argument('--http-proxy',
-                                 help='HTTP proxy URL')
-        self.parser.add_argument('--release-notes-repo-url',
-                                 help='Release notes repository URL')
-        self.parser.add_argument('--release-notes-repo-branch',
-                                 help='Branch of the release notes repository '
-                                 'to checkout')
-        self.parser.add_argument('--push-repo-url',
-                                 help='URL of the repository used for pushing')
-        self.parser.add_argument('--push-repo-branch',
-                                 help='Branch of the repository used for '
-                                 'pushing',
-                                 default='master')
-        self.parser.add_argument('--committer-name',
-                                 help='Name used when creating a commit and '
-                                 'bumping spec files')
-        self.parser.add_argument('--committer-email',
-                                 help='Email used when creating a commit and '
-                                 'bumping spec files')
+        self._add_subparser()
+
+    def _add_subparser(self):
+        subparser = self.parser.add_subparsers(
+            dest="subcommand",
+            help="Available subcommands")
 
+        for command, help_msg, arg_groups in SUBCOMMANDS:
+            parser_command = subparser.add_parser(command, help=help_msg)
+            for arg_group in arg_groups:
+                for arg, options in arg_group.items():
+                    arg = arg.split()
```

Comment:
If you follow my tuple suggestion, remove this line.
Also, rename to `args`, since there may be more than one.